prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -87,6 +95,40 @@ object FhirPathUtils : Logging {
         bundle: Bundle,
         expressionNode: ExpressionNode
     ): String {
-        return defaultPathEngine.evaluateToString(appContext, focusResource, bundle, bundle, expressionNode)
+        val evaluated = defaultPathEngine.evaluate(appContext, focusResource, bundle, bundle, expressionNode)
+        if (evaluated.size == 0) return ""
```

Comment:
Having multiple returns in a function is not good practice.  Try to refactor this one to have one return.

Reply:
Interesting, I've always been taught that returning early when conditions to continue have not been met is a good thing

Reply:
Short circuiting logic that doesn't need to be run is a good thing, but having multiple returns is not since it makes it more difficult to debug and can result in spaghetti code. Having an if statement that short circuits/skips logic and goes directly to a single return is a good way to handle this.

Reply:
I tried to refactor this to both be explicit that an empty evaluated list would return an empty string, only have one return, and better chain the if statements.

To be honest I don't like what I came up with so I'm open to more feedback.